scripts/build/termux_step_install_license.sh

```diff
@@ -0,0 +1,21 @@
+termux_step_install_license() {
+	mkdir -p "$TERMUX_PREFIX/share/$TERMUX_PKG_NAME"
+	local COUNTER=0
+	for LICENSE in $(echo $TERMUX_PKG_LICENSE | sed 's/,/ /g'); do
+		if [ -f "$TERMUX_SCRIPTDIR/packages/termux-licenses/LICENSES/${LICENSE}.txt" ]; then
+			if [[ $COUNTER > 0 ]]; then
+				ln -sf "../LICENSES/${LICENSE}.txt" "$TERMUX_PREFIX/share/$TERMUX_PKG_NAME/LICENSE.${COUNTER}"
+			else
+				ln -sf "../LICENSES/${LICENSE}.txt" "$TERMUX_PREFIX/share/$TERMUX_PKG_NAME/LICENSE"
+			fi
+		else
+			for POSSIBLE_LICENSE in "LICENSE*" "license*" "COPYRIGHT" "copyright"; do
+				if [ -f "$TERMUX_PKG_SRCDIR/$POSSIBLE_LICENSE" ]; then
+					cp "$TERMUX_PKG_SRCDIR/$POSSIBLE_LICENSE" "$TERMUX_PREFIX/share/$TERMUX_PKG_NAME/"
+					break
```

Comment:
@Grimler91 Maybe it will be better to override `termux_step_install_license()` when license is custom ? Some packages (e.g. frobtads) have their license in subdir.

Reply:
We could also create a new variable `TERMUX_PKG_LICENSE_FILE` which, if set, is copied to `$PREFIX/share/$PKG`